solidity/test/bridge/VendingMachine.test.ts

```diff
@@ -28,33 +36,28 @@ describe("VendingMachine", () => {
       tokenHolder,
       thirdParty,
     ] = await ethers.getSigners()
+  })
 
-    const TestERC20 = await ethers.getContractFactory("TestERC20")
-    tbtcV1 = await TestERC20.deploy()
-    await tbtcV1.deployed()
-
-    const TBTC = await ethers.getContractFactory("TBTC")
-    tbtcV2 = await TBTC.deploy()
-    await tbtcV2.deployed()
-
-    await tbtcV1.mint(tokenHolder.address, initialBalance)
+  beforeEach("load test fixture", async () => {
+    const contracts = await waffle.loadFixture(vendingMachineDeployment)
+    tbtcV1 = contracts.tbtcV1 as TestERC20
+    tbtcV2 = contracts.tbtcV2 as TBTC
+    vendingMachine = contracts.vendingMachine as VendingMachine
 
-    const VendingMachine = await ethers.getContractFactory("VendingMachine")
-    vendingMachine = await VendingMachine.deploy(
-      tbtcV1.address,
-      tbtcV2.address,
-      unmintFee
-    )
-    await vendingMachine.deployed()
+    await tbtcV1.mint(await tokenHolder.getAddress(), initialBalance)
 
-    await vendingMachine.connect(deployer).transferOwnership(governance.address)
     await vendingMachine
       .connect(deployer)
-      .transferUnmintFeeUpdateInitiatorRole(unmintFeeUpdateInitiator.address)
+      .transferOwnership(await governance.getAddress())
+    await vendingMachine
+      .connect(deployer)
+      .transferUnmintFeeUpdateInitiatorRole(
+        await unmintFeeUpdateInitiator.getAddress()
+      )
     await vendingMachine
       .connect(deployer)
       .transferVendingMachineUpgradeInitiatorRole(
-        vendingMachineUpgradeInitiator.address
+        await vendingMachineUpgradeInitiator.getAddress()
```

Comment:
Would be good to agree on one solution across all repositories cc @dimpar @lukasz-zimnoch @nkuba. Another approach is [here](https://github.com/keep-network/keep-core/blob/874e247e8f21cec976f93b38604c37a1f77b8872/solidity/random-beacon/test/RandomBeacon.Relay.test.ts#L6). I think the one presented by @tomaszslabon in this PR is "cleaner" because we do not have to dig too deep into libraries used under the hood although it does require some boilerplate code.